### ISSUE TITLE
4.x regression on Safari: TransactionInactiveError

### ISSUE THREAD

**Issue by mustafa0x** [jakearchibald/idb-keyval#99]
`Unhandled Promise Rejection: TransactionInactiveError: Failed to execute 'get' on 'IDBObjectStore': The transaction is inactive or finished.`

Still debugging this, but seems to be happening. Edit: I rolled back to 3.2.0 and the bug stopped occurring.

**Comment by jakearchibald** [comment]
Can you give me steps to reproduce the error?

**Comment by mustafa0x** [comment]
```html
<script src=https://unpkg.com/idb-keyval@4.0.1/dist/iife/index-min.js></script>
<script type=module>
function debounce(fn, timeout) {
    let timeout_id;
    return (...args) => {
        clearTimeout(timeout_id);
        timeout_id = setTimeout(() => fn(...args), timeout);
    };
}
const update_debounced = debounce(async () => {
    (await idbKeyval.get('foo'));
}, 50);

async function init() {
    (await idbKeyval.get('bar'));
    update_debounced();
}
init();
</script>
```

Replacing with `@3.2.0/dist/idb-keyval-iife.min.js` stops the error.

**Comment by jakearchibald** [comment]
Thanks, I can reproduce this

**Comment by jakearchibald** [comment]
Fixed in 1af0a00, released as 5.0.0. Thanks for reporting this!
